http/1.6/modules/ROOT/pages/http-request-ref.adoc

```diff
@@ -82,104 +204,226 @@ For the particular case of the HTTP connector, this means that it's only a good
+		</http:request>
+	</flow>
+----
 
-The following elements *don't* apply to HTTP responses:
+The HTTP *Request* operation sends a POST request to the host location. The HTTP request contains `Content-Type: application/x-www-form-urlencoded` and the body is `key1=value1&key2=value2`.
 
-* Query parameters
-* URI parameters
-* Inbound attributes related to the HTTP request URI
 
-In addition, the HTTP request operation adds the following attributes to the Mule message when receiving a response:
+== Configure Round-Robin Requests
 
-* `attributes.statusCode`: Status code of the HTTP response
-* `attributes.reasonPhrase`: Reason phrase of the HTTP response
+The HTTP *Request* operation connects to configured hosts using round robin DNS. Mule resolves all IP addresses associated to the specified host and performs load balancing by distributing the requests across all returned IPs.
 
-== Round-Robin Requests
+When the operation connects to resources that require authentication, the external service needs to replicate session information between IP addresses under the host of your service. Otherwise, the HTTP requests might get rejected for being unauthorized.
 
-The request operation connects to configured hosts using round robin DNS. Mule runtime engine resolves all IP addresses associated to the specified host and performs load balancing by distributing the requests across all returned IPs.
+If the external resource does not manage sticky sessions, add the service host name to the `mule.http.disableRoundRobin` system property. The HTTP request won't use round robin DNS to connect to the configured host.
 
-When connecting to resources that require authentication, the external service needs to replicate session information between IP addresses under the host of your service. Otherwise, your requests might get rejected for being unauthorized.
+To configure this system property, set it when starting Mule by adding: +
+`./mule -M-Dmule.http.disableRoundRobin=serverhostname.com`
 
-When your external resource does not handle sticky sessions you need to add the service host name to the `mule.http.disableRoundRobin` system property when starting the Mule Runtime:
+== Configure Response Validator
 
-[source,console]
-----
-./mule -M-Dmule.http.disableRoundRobin=serverhostname.com
-----
+When the HTTP *Request* operation receives an HTTP response, the operation validates the response through its status code. By default, HTTP Connector throws an HTTP error response when the status code is higher than or equal to 400. Consequently, if the server returns a 404 (Resource Not Found) or a 500 (Internal Server Error) a failure occurs and the connector triggers the error handling.
+
+Two ASCII `..` full stop characters define the range of failure status codes. Any value between 500 and 599 is considered a failure and raises an error. If the HTTP response has any other status code value, it's considered a success.
```

Comment:
```suggestion
Two ASCII full-stop characters (`..`) define the range of failure status codes. Any value between 500 and 599 is considered a failure and raises an error. If the HTTP response has any other status code value, validation is considered a success.
```
"between" = from 500 through 599? clarify what is included.
raise an error vs. throw an error?

Reply:
Is it confusing that the previous text included values 400 and above and this text 500 through 599?